src/game/server/swarm/asw_player.h

```diff
@@ -244,6 +244,7 @@ class CASW_Player : public CBaseMultiplayerPlayer, public IASWPlayerAnimStateHel
 	// status of selecting marine/weapons in the briefing
 	CNetworkVar( int, m_nChangingMR );
 	CNetworkVar( int, m_nChangingSlot );
+	CNetworkVar(Vector, m_vecLobbyLaserColor );
```

Comment:
```suggestion
	CNetworkVector( m_vecLobbyLaserColor );
```

are laser colors per-player or per-marine?

Reply:
In lobby, I have it programmed for each human player to be able to set their own laser colors.

AI currently just choose a random color along the H and S spectrum.

Reply:
In game, each marine has it's own color netprop.